### PR TITLE
added test descriptions for progress bar 

### PR DESCRIPTION
**Pull request by FursovaKate**
and fix test failures due to the browser size

### CODE REVIEW

File: jdi-light-angular-tests/src/test/resources/test.properties [hunk 1]
```diff
@@ -14,7 +14,7 @@ log.level=INFO
 #assert.type=strict | soft
 
 #page.load.strategy=normal
-#browser.size=MAXIMIZE | 1024x762
+browser.size= 1024x762
```
Comment (pnatashap) [inline jdi-light-angular-tests/src/test/resources/test.properties:17]: мы не будем менять размер окна для всего запуска, будет масса нестабильностей

**Review by pnatashap — CHANGES_REQUESTED** [review]

**Review by MayaElf — APPROVED** [review]

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: jdi-light-angular-tests/src/main/java/io/github/com/StaticSite.java [hunk 2]
```diff
@@ -9,15 +9,19 @@
...
 import com.jdiai.tools.DataClass;
 import io.github.com.pages.AngularPage;
+import io.github.com.pages.ProgressBarPage;
 
-@JSite("https://jdi-testing.github.io/jdi-light/")
+@JSite("https://jdi-testing.github.io/jdi-light//angular-page/#/")
 public class StaticSite {
 
-    public static final User DEFAULT_USER = new User();
+    public static User DEFAULT_USER = new User();
```
[the remark this answers is not in the document]
Comment (alla-blinkova) [reply jdi-light-angular-tests/src/main/java/io/github/com/StaticSite.java:18]: I guess it should be renamed to `defaultUser`

[the remark this answers is not in the document]
Comment (pnatashap) [reply jdi-light-angular-tests/src/main/java/io/github/com/StaticSite.java:18]: Actually you do not need this User at all, now Angular does not use login from the main site

File: jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/ProgressBarUnitTests.java [hunk 3]
```diff
@@ -4,19 +4,31 @@
...
+import static io.github.com.pages.ProgressBarPage.showBufferProgressBarButton;
+import static io.github.com.pages.ProgressBarPage.showIndeterminateProgressBarButton;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
 
 public class ProgressBarUnitTests extends TestsInit {
     @BeforeMethod
     public void before() {
         skipForFirefox();
```
Comment (pnatashap) [inline jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/ProgressBarUnitTests.java:28]: skipForFirefox also should be removed

File: jdi-light-angular-tests/src/test/resources/test.properties [hunk 4]
```diff
@@ -5,10 +5,10 @@ domain=https://jdi-testing.github.io/jdi-light/
 page.check.after.open=NONE
 # NEW_PAGE | EVERY_PAGE
 #element.search.strategy=strict | soft | visible, multiple | any, single
-element.search.strategy=strict
+element.search.strategy=soft
```
Comment (pnatashap) [inline jdi-light-angular-tests/src/test/resources/test.properties:8]: Why do we need to change it?

File: jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/enums/ProgressBarColors.java [hunk 5]
```diff
@@ -3,7 +3,10 @@
 public enum ProgressBarColors {
     BLUE("mat-primary"),
     YELLOW("mat-accent"),
-    RED("mat-warn");
+    RED("mat-warn"),
```
Comment (pnatashap) [inline jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/enums/ProgressBarColors.java:6]: Why do we keep old values?

**Review by pnatashap — CHANGES_REQUESTED** [review]

**Pull request merged**